cdap-docs/reference-manual/source/cli-api.rst

```diff
@@ -285,6 +285,10 @@ These are the available commands:
    ``truncate dataset instance <dataset-name>``,"Truncates a Dataset."
    **Explore**
    ``execute <query> [<timeout>]``,"Executes a Query with optional <timeout> in minutes (default is no timeout)."
+   **Metrics**
+   ``get metric value <metric-name> [<tags>] [start <start>] [end <end>]``,"Gets the value of a metric. Provide <tags> as a map in the format 'a=b, c=d'."
```

Comment:
Why is the <tags> map format different for this and the following two lines? (same inconsistency in the string the java code returns).